packages/charts/src/styles/vaadin-chart-base-styles.js

```diff
@@ -149,1048 +149,1248 @@ export const chartStyles = css`
+      font-weight: bold;
+    }
 
-  :where([styled-mode]) .highcharts-range-label text {
-    fill: var(--_secondary-label);
-  }
+    .highcharts-button-disabled text {
+      fill: var(
+        --highcharts-neutral-color-80,
+        var(--vaadin-charts-button-disabled-label, var(--vaadin-color-disabled))
+      );
+    }
 
-  :where([styled-mode]) .highcharts-range-input rect {
-    fill: var(--vaadin-charts-range-input-background, var(--vaadin-background-container));
-    rx: var(--vaadin-charts-button-border-radius, var(--vaadin-radius-m));
-    ry: var(--vaadin-charts-button-border-radius, var(--vaadin-radius-m));
-    transition: fill 250ms;
-  }
+    .highcharts-range-selector-buttons .highcharts-button {
+      stroke-width: 0;
+    }
 
-  :where([styled-mode]) .highcharts-range-input:hover rect {
-    fill: var(--vaadin-charts-range-input-background-hover, var(--vaadin-background-container));
-  }
+    .highcharts-range-label rect {
+      fill: none;
+    }
 
-  :where([styled-mode]) .highcharts-range-input text {
-    fill: var(--_data-label);
-  }
+    .highcharts-range-label text {
+      fill: var(--highcharts-neutral-color-60, var(--_secondary-label));
+    }
 
-  :where([styled-mode]) input.highcharts-range-selector {
-    position: absolute;
-    border: 0;
-    width: 1px;
-    /* Chrome needs a pixel to see it */
-    height: 1px;
-    padding: 0;
-    text-align: center;
-    left: -9em;
-    /* #4798 */
-  }
+    .highcharts-range-input rect {
+      fill: var(--vaadin-charts-range-input-background, var(--vaadin-background-container));
+      rx: var(--vaadin-charts-button-border-radius, var(--vaadin-radius-m));
+      ry: var(--vaadin-charts-button-border-radius, var(--vaadin-radius-m));
+      transition: fill 250ms;
+    }
 
-  :where([styled-mode]) .highcharts-crosshair-label text {
-    fill: var(--_bg);
-    font-size: 1.1em;
-  }
+    .highcharts-range-input:hover rect {
+      fill: var(--vaadin-charts-range-input-background-hover, var(--vaadin-background-container));
+    }
 
-  :where([styled-mode]) .highcharts-crosshair-label .highcharts-label-box {
-    fill: inherit;
-  }
+    .highcharts-range-input text {
+      fill: var(--highcharts-neutral-color-80, var(--_data-label));
+      font-size: 1em;
+    }
 
-  :where([styled-mode]) .highcharts-candlestick-series .highcharts-point {
-    stroke: var(--vaadin-charts-candlestick-line, var(--vaadin-border-color));
-    stroke-width: 1px;
-  }
+    .highcharts-range-input {
+      stroke-width: 1px;
+      stroke: var(--highcharts-neutral-color-20);
+    }
 
-  :where([styled-mode]) .highcharts-candlestick-series .highcharts-point-up {
-    fill: var(--_color-positive);
-  }
+    input.highcharts-range-selector {
+      position: absolute;
+      border: 0;
+      width: 1px; /* Chrome needs a pixel to see it */
+      height: 1px;
+      padding: 0;
+      text-align: center;
+      left: -9em; /* #4798 */
```

Comment:
This could be probably removed? The issue is fixed https://github.com/highcharts/highcharts/issues/4798.